src/Components/HomeSections/Blog/Blog.module.scss

```diff
@@ -0,0 +1,27 @@
+@import '../../../styles/colors.scss';
```

Comment:
is it possible to make the import url inside scss file also absolute path?

Reply:
Yes it is